extensions/2.0/Khronos/KHR_texture_ktx/README.md

```diff
@@ -0,0 +1,293 @@
+VK_FORMAT_R32G32B32A32_SFLOAT  
+
+
+- **RGB:** A texture that uses red, green, and blue channels. Alpha channel is unused and not sampled at runtime.  
+
+VK_FORMAT_E5B9G9R9_UFLOAT_PACK32  
+VK_FORMAT_B10G11R11_UFLOAT_PACK32  
+VK_FORMAT_R16G16B16_SFLOAT  
+VK_FORMAT_R16G16B16_UNORM  
+VK_FORMAT_R32G32B32_SFLOAT  
+
+- **RG_** A texture that uses, red and green channels. All other channels are unused and their values are not sampled at runtime.  
+
+VK_FORMAT_R16G16_SFLOAT  
+VK_FORMAT_R16G16_UNORM  
+VK_FORMAT_R32G32_SFLOAT  
+
+
+- **Red:** A texture that uses only red channel. All other channels are unused and their values are not sampled at runtime.  
+
+VK_FORMAT_R16_SFLOAT  
+VK_FORMAT_R16_UNORM  
+VK_FORMAT_R32_SFLOAT  
+
+**Implementation Notes**  
+This section is non-normative
+
+This extension defines a number of formats that are supported for images used as texture sources.  
+It does not define how these formats are stored internally, it is up to implementations to use a suitable texture format.  
+The source texture shall be used in shader (BRDF) calculations in a way that retains the increased range.  
+However, this extension does not define the way pixel data is written to framebuffer or how swapchain is presented using the increased range.  
+
+If ktx image 
+
+## KTX header fields
+
+`supercompressionScheme` MUST be 3 (ZLIB).  
+
+- vkFormat must be one of:  
+VK_FORMAT_E5B9G9R9_UFLOAT_PACK32  
+VK_FORMAT_B10G11R11_UFLOAT_PACK32  
+VK_FORMAT_R16G16B16A16_SFLOAT  
+VK_FORMAT_R16G16B16A16_UNORM  
+VK_FORMAT_R32G32B32A32_SFLOAT  
+VK_FORMAT_R16G16B16_SFLOAT  
+VK_FORMAT_R16G16B16_UNORM  
+VK_FORMAT_R32G32B32_SFLOAT
+VK_FORMAT_R16G16_SFLOAT  
+VK_FORMAT_R16G16_UNORM  
+VK_FORMAT_R32G32_SFLOAT  
+VK_FORMAT_R16_SFLOAT  
+VK_FORMAT_R16_UNORM  
+VK_FORMAT_R32_SFLOAT  
+
+
+
+### Additional requirements
+
+Regardless of the format used, these additional restrictions apply for compatibility reasons:
+
+- Swizzling metadata (`KTXswizzle`) MUST be `rgba` or omitted.
+- Orientation metadata (`KTXorientation`) MUST be `rd` or omitted.
+
+- Color space information in the DFD MUST match the expected usage, namely:
+  - For textures with **color data** (e.g., base color maps),
+    - `colorPrimaries` MUST be `KHR_DF_PRIMARIES_BT709`;
+    - `transferFunction` MUST be `KHR_DF_TRANSFER_LINEAR`.
+  - For textures with **non-color data** (e.g., normal maps),
+    - `colorPrimaries` MUST be `KHR_DF_PRIMARIES_UNSPECIFIED`;
+    - `transferFunction` MUST be `KHR_DF_TRANSFER_LINEAR`.
+
+  > **Implementation Note**: Engines may need to generate missing mipmaps at runtime when a full mip pyramid is not available.
+
+
+### Using KTX v2 Images with Basis Universal Supercompression for Material Textures
+
+When a texture referencing a KTX v2 image with Basis Universal supercompression is used for glTF 2.0 material maps (both color and non-color), the KTX v2 image MUST be of **2D** type as defined in the KTX v2 Specification, Section 4.1.
```

Comment:
This statement contradicts the supercompression restriction above.

Reply:
This is a cut & paste error - it should read:
Using KTX v2 Images for Material Textures